google-cloud-logging/lib/google/cloud/logging/sink/list.rb

```diff
@@ -71,7 +71,7 @@ def next
             return nil unless next?
             ensure_service!
             list_grpc = @service.list_sinks token: token, max: @max
-            self.class.from_grpc list_grpc, @service
+            self.class.from_grpc list_grpc, @service, @max
```

Comment:
Want to change the `max` param to keyword arg to match `Entry::List` and `Log::List`?